test/TestExpectations.json

```diff
@@ -1323,6 +1323,55 @@
     "expectations": ["FAIL"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
+  {
+    "testIdPattern": "[elementhandle.spec] ElementHandle specs ElementHandle.touchEnd should work",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["cdp", "firefox"],
+    "expectations": ["FAIL"],
+    "comment": "Same reason as '[touchscreen.spec] *'"
```

Comment:
It would be better to copy paste them rather then referencing as we my remove the other at some point.

Reply:
I opted for this because the reason in the other one is "TODO add a comment (etc.)", but I'll copy-paste that :)

Reply:
Yeah, we added the field well into this file so not all is filled but its good to know that its not been investigated why it failed.